kafka-ui-react-app/src/components/Topics/List/List.tsx

```diff
@@ -4,6 +4,7 @@ import Breadcrumb from 'components/common/Breadcrumb/Breadcrumb';
 import { NavLink } from 'react-router-dom';
 import { clusterTopicNewPath } from 'lib/paths';
 import ListItem from './ListItem';
+import _ReadOnlyContext from '../../contexts/ReadOnlyContext';
```

Comment:
Let's use proper naming without `_`. You can use destructuring assignment to extract `isReadOnly` flag without another context variable.